CMakeLists.txt

```diff
@@ -158,13 +158,24 @@ if (XSDK_ENABLE_Fortran)
    enable_language(Fortran)
 endif()
 
-# Suppress warnings about MACOSX_RPATH
-set(CMAKE_MACOSX_RPATH OFF CACHE BOOL "")
+set(CMAKE_MACOSX_RPATH ON CACHE BOOL "")
+
+if (APPLE)
+   # Set policy CMP0068 to NEW so that install_name will include RPATH settings
+   if(POLICY CMP0068)
+       cmake_policy(SET CMP0068 NEW)
+   endif()
+
+   set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+endif()
 
 # CMake needs to know where to find things
 set(MFEM_CMAKE_PATH ${PROJECT_SOURCE_DIR}/config)
 set(CMAKE_MODULE_PATH ${MFEM_CMAKE_PATH}/cmake/modules)
 
+# Load standard install paths
```

Comment:
Shouldn't this line come after `include(GNUInstallDirs)`? So that we ensure that our "custom" path overrides the default value set by GNUInstallDirs?

Reply:
No, because this is a cache variable, so we need to leapfrog GNUInstallDirs before it sets it, at which point it'll be too late for us to override it, in other words: doing set on a cache variable doesn't override it, just sets it if undefined.

Reply:
We could use you to train INL's AI tool